docs/AGREEMENT.md

```diff
@@ -39,3 +39,17 @@ I promise to provide valuable critique necessary for the improvement of the proj
 I plan to take an active role in learning and contributing to all areas of the project. I will ensure to collaborate with the team on every deliverables required for the goals, sprints and milestone for this project and for the class.
 
 If conflict arises, I will ensure that all perspectives are heard and work on a solution-based resolution.
+
+
+### Juan's Agreement
+I, Juan (Does it have to be this formal?), agree to the following:
+
```

Comment:
Markdown also might render funny if there are blank lines between list items. Can you remove them? E.g.:

```
1. foo
2. bar
3. baz
```